yet_another_imod_wrapper/fiducials.py

```diff
@@ -48,13 +50,14 @@ def align_tilt_series_using_fiducials(
         fiducial_size=fiducial_size,
         rotation_angle=nominal_rotation_angle
     )
-    run_batchruntomo(
-        directory=output_directory,
-        basename=basename,
-        directive=directive
-    )
-    if etomo_output.contains_alignment_results is False:
-        raise RuntimeError(f'{basename} failed to align correctly.')
+    if etomo_output.contains_alignment_results is False or skip_if_completed is False:
```

Comment:
something feels off about the logic here, please check in AT too - 

etomo output contains alignment results=False, skip_if_completed=False -> run
etomo output contains alignment results=True, skip_if_completed=False -> run
etomo output contains alignment results=True, skip_if_completed=True -> don't run
etomo output contains alignment results=False, skip_if_completed=True -> don't run

The last one is wrong, no?

Reply:
I think the logic is correct: if contains alignment results=False, we always want it to run as that TS hasn't been run yet, and if contains alignment results=False then it always will. If skip_if_completed is False then it will run (as we want it to do for all TS as we want it to re-run for ones already run). If skip_if_completed is True then for TS it has already run on it will skip (contains will be True for these) and if False it will run.

This works as this function is run for each individual TS, no?

Reply:
you're totally right, my bad, had a hard time with all the negatives flying around - thanks for the patience!